internal/controller/argocd.go

```diff
@@ -129,9 +135,11 @@ func (clctrl *ClusterController) InitializeArgoCD() error {
 				8080,
 				argoCDStopChannel,
 			)
+
 			argoCDToken, err = argocd.GetArgoCDToken("admin", argocdPassword)
+
 			if err != nil {
-				return err
+				return fmt.Errorf("Error getting argoCDToken : %w", err)
```

Comment:
It's a bit of a norm in Go that error messages returned should not be capitalized the first letter. That's so the caller can choose to print that, so if you have something like your message, it'll render as:

```
Error: Error getting argoCDToken : foo bar baz
```

Without capitalization, it becomes more readable:

```
Error: error getting argoCDToken : foo bar baz
```

One of the linters we bring will have this check done automatically!

Reply:
Done  :)